slsa-provenance/main.go

```diff
@@ -116,12 +120,14 @@ func main() {
 			panic(err)
 		}
 
-		fmt.Printf(string(attBytes))
-		fmt.Printf(`::set-output name=provenance::%s`, string(attBytes))
+		filename := fmt.Sprintf("%s.intoto", binary)
+		if err := ioutil.WriteFile(filename, attBytes, 0600); err != nil {
+			panic(err)
+		}
+		fmt.Printf(`::set-output name=provenance-name::%s`, filename)
```

Comment:
what's the reasoning for having the print here rather than in the shell of the re-usable workflow? I'm not against it, just curious.

Reply:
removed this since now we create/sign in the same step, so no need for separating.

on the other hand we still output the name of the signed provenance. only because that way the workflow doesn't change if we want to change this